torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10487,6 +10487,9 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                    skips=(
                        # Skip since real and imag don't have out variants.
                        DecorateInfo(unittest.expectedFailure, 'TestUnaryUfuncs', 'test_out_arg_all_dtypes'),
+                       # numpy() raises TypeError: Got unsupported ScalarType ComplexHalf
```

Comment:
This is a little surprising because I thought I updated the tests to map complexhalf to complexfloat for NumPy, but maybe I missed something?

Reply:
I think it is a recent change (probably PR was based on older master) and there is probably some other failure now. Let me check. Thanks!

Reply:
You are right. It works now.